README.md

```diff
@@ -22,7 +22,7 @@ return <Video src={myVideo} />;
 ```bash
 cd your-next-app
 npm install @mux/next-video
-npx @mux/next-video init
+npx next-video init
```

Comment:
I wouldn't do this, it might work but only because you installed `@mux/next-video` first.

`npx @mux/next-video init` will always work with the valid package name.
I might run that first if I go from memory instead of following the readme.